### DISCUSSION TITLE
Can't access mobile files

### DISCUSSION THREAD

**Discussion by GeneralAccess** [elementary/files#2352] [Q&A] [answered]
I've installed elementaryOS on a spare disk to try it out and perhaps switch from my current OS. It works OK apart from connecting to my mobile. I get an "empty directory" (or similar) message on my desktop trying to access files on my Samsung Android-6 mobile linked with a USB cable (I have given permission on the mobile). It works with my Huawei (which doesn't ask).

**Comment by jeremypw** [comment]
A couple of things to check (I have Android 14 so I am sure of the exact UI for Android 6 - you can Google it)

 - Use the manufacturer supplied USB cable - not all cables work for file transfer.
 - Check USB settings on phone to make sure you have file transfer enabled (with the mtp protocol)
 - When you plug in the phone you should get a notification of some kind to allow/select file transfer instead of charging (or you may need to go into USB on the pull down settings.
 - *Refresh* the view on Files by pressing `F5` or the refresh button.

File transfer is working for me with Elementary 7 and an Android 14 phone.

**Comment by jeremypw** [comment]
You can also transfer files over Bluetooth.

**Reply by GeneralAccess** [reply]
I know that some USB cables are only for charging, I don't use a manufacturer supplied cable, but I know which of my cables can be used for data.
Otherwise I do all those things every time on my current OS. I have done exactly as normal, in fact used the same cable and procedure for the Samsung and the Huawei. The only difference is that the Huawei doesn't ask for permission. Perhaps there is a bug relating to permissions.
I have never made Bluetooth work on Linux. Even if I could get that to work, I wouldn't want to rely on it. I would need the basic cable method to work before switching to elementaryOS.

**Comment by jeremypw** [comment, marked as answer]
Hmm, not sure what the problem is then. I got file transferring working both on my new Android 14 phone and an older Android 8 phone.  I assume you are using Elementary 6.1 or 7?  What is the path you get if you click on the empty part of the pathbar?  It should be something like `mtp://SAMSUNG_SAMSUNG_Android_XXXXXXXXXXX`.  There were some problems with MTP on older versions of Files but it seems to work now.

**Reply by GeneralAccess** [reply]
It's the latest one, which I downloaded yesterday - 7.1. I get a similar reference to you, but without the beginning:  SAMSUNG_SAMSUNG_Android_XXXXXXXXXXX

Later:
Fiddling about with the cursor trying to show the whole of that reference triggered something and the files now show! I've just spotted a circle with an arrow at the right hand end of the field, maybe that's what does it. I don't think my current OS has got that. Sorry for wasting your time by not spotting it before. However you have fixed Bluetooth for me which is a definite win!
I'll dabble with elementaryOS for a while before deciding whether to switch.
Thanks.

**Reply by jeremypw** [reply]
Glad you found a solution! You should not have to press the "navigate to path" button (unless you edited the path), but if it works then good!  When the pathbar is not focused then pressing the button with the circular arrow at the end of the pathbar forces Files to reload the location. It sounds like your phone is not in "MTP transfer mode" if you are not getting the `mtp://` at the start of the path.  There should be a way of putting your phone in that mode somewhere in the settings.

**Reply by GeneralAccess** [reply]
(It was in MTP transfer mode)

**Comment by GeneralAccess** [comment]
I've just tried Bluetooth and it worked first time from desktop to mobile. **Impressive!** I haven't worked out how to do it the other way round, but that is a phone problem for me to work out.
